memory/player_party_manager.py

```diff
@@ -0,0 +1,39 @@
+import memory.core as Memory
+from helpers.vec import Vec
+
+class PlayerPartyManager():
+    def __init__(self):
+        self.memory = Memory.mem_handle()
+        self.base = None
+
+    def update(self):
+        self.memory.update()
+
+        if self.memory is not None:
+            offset = [
+                0xB8, 0x10, 0x40, 0x30, 0x10, 0x38, 0xA8, 0x10, 
+                0xC8, 0x38, 0xE8, 0x20, 0x40, 0xD0, 0x200
+            ] 
+            ptr = self.memory.get_pointer(0x2EAA3D0, offset)
```

Comment:
Prefer putting this as a named constant (the address).

Reply:
just noting here - but i had an issue where moving it as a constant may see it show up initialized as an empty list at times, causing issues.